README.md

```diff
@@ -100,7 +101,8 @@ To enrich the server, certain Maven profiles can be activated:
 
 | Profile Group                           | Profile Name             | Description                                                                     | Parameters                                                                  | Further Info                                                                                                                        |
 |-----------------------------------------|--------------------------|---------------------------------------------------------------------------------|-----------------------------------------------------------------------------|-------------------------------------------------------------------------------------------------------------------------------------|
-| **HTTP Endpoints (Fetch/Ingestion)**    | http-ingest              | Enables a HTTP endpoint for to insert LDES members.                             | [HTTP configuration](#example-http-ingest-fetch-configuration)              | Endpoint:<br><br>- URL: /{ldes.collection-name}<br>- Request type: POST<br>- Accept: "application/n-quads", "application/n-triples" |
+| **HTTP Ingest v2**                      | http-ingest-v2           | Enables a HTTP endpoint to insert LDES members.                                 | [HTTP configuration](#example-http-ingest-fetch-configuration)              | Endpoint:<br><br>- URL: /{ldes.collection-name}<br>- Request type: POST<br>- Accept: "application/n-quads", "application/n-triples" |
```

Comment:
Is there a need to still call it ingest v2 if its on the epic branch?

Reply:
The profile http-ingest still exists as you need it to seed the old 'members' collection which is still required for all other functionalities such as fragmentation, snapshotting, retention, etc.
I removed the v2 from the first column but kept it in the second column to make sure the profile names match. We will have to update those at the end of implementing server v2 when removing the old ingest.